client/packages/common/src/ui/forms/JsonForms/components/DateTime.tsx

```diff
@@ -63,22 +73,39 @@ const UIComponent = (props: ControlProps) => {
         <FormLabel sx={{ fontWeight: 'bold' }}>{label}:</FormLabel>
       </Box>
       <Box flexBasis={FORM_INPUT_COLUMN_WIDTH}>
-        <BaseDateTimePickerInput
-          // undefined is displayed as "now" and null as unset
-          value={data ?? null}
-          onChange={e => {
-            try {
-              setError('');
-              if (e) handleChange(path, e.toISOString());
-            } catch (err) {
-              setError((err as Error).message);
-              console.error(err);
-            }
-          }}
-          inputFormat="dd/MM/yyyy hh:mm"
-          error={error || props.errors}
-          // readOnly={!!props.uischema.options?.['readonly']}
-        />
+        {!dateOnly ? (
+          <BaseDateTimePickerInput
+            // undefined is displayed as "now" and null as unset
+            value={data ?? null}
+            onChange={e => {
+              try {
+                setError('');
+                if (e) handleChange(path, e.toISOString());
+              } catch (err) {
+                setError((err as Error).message);
+                console.error(err);
+              }
+            }}
+            inputFormat="dd/MM/yyyy hh:mm"
+            error={error || props.errors}
+            // readOnly={!!props.uischema.options?.['readonly']}
```

Comment:
@clemens-msupply the "readOnly" prop was already commented out. Should we re-enable it? (Seems reasonable to me, but you may have had a good reason for disabling?)

Reply:
No I can't remember why this was done, think it can be enabled again!

Reply:
Cool, done 9dc2803a8